### PR TITLE
Feat/claimed and unclaimed graphs

### LINKED ISSUE #74: Total Unclaimed and Claimed Funds
- [x] Create a pie chart showing claimed and unclaimed funds for each election and for all elections
- [x] Per delegate and for all delegates
- [ ] Show list of delegates with unclaimed funds and amounts

https://recharts.org/en-US/examples/CustomActiveShapePieChart

image

### LINKED ISSUE #99: Delegate Claimed and Unclaimed Stacked Bar
A stacked bar graph showing the amount of claimed and unclaimed funds for each delegate

image

- [x] Add the ability to filer to show unclaimed, claimed or both (both by default)
- [x] Inclaimed can be a light grey for now

### LINKED ISSUE #126: Order elements by date
Order exchange rate by date

image
image

### LINKED ISSUE #127: Create a graph by delegate with dates for the exchange rate
image

image

### PR DESCRIPTION
**Pull request by JAMares**
### What does this PR do?

- Resolve #99 
- Resolve #74
- Resolve #126 
- Resolve #127 

### Steps to test

1. Run project
2. Go to Income
3. You should see this:
imagen

#### CheckList

- [X] Follow proper Markdown format
- [X] The content is adequate
- [X] The content is available in both english and spanish
- [X] I Ran a spell check

### CODE REVIEW

**Comment by JAMares** [comment]
This PR is from the new branch that work fine

File: .vscode/settings.json [hunk 1]
```diff
@@ -7,5 +7,6 @@
     "**/.yarn": true,
     "**/.pnp.*": true
   },
-  "prettier.prettierPath": "./webapp/.yarn/sdks/prettier/index.js"
+  "prettier.prettierPath": "./webapp/.yarn/sdks/prettier/index.js",
+  "eslint.workingDirectories": [{ "mode": "auto" }]
```
Comment (leisterfrancisco) [inline .vscode/settings.json:11]: general knowledge

Why this?

File: webapp/src/gql/income.gql.js [hunk 2]
```diff
@@ -90,3 +90,40 @@ export const GET_TOTAL_INCOME_BY_ELECTIONS_QUERY = gql`
...
+export const GET_TOTAL_CLAIMED_AND_UNCLAIMED = gql`
+  query getTotalClaimedAndUnclaimed {
+    total_claimed_and_unclaimed {
+      amount
+      category
+      usd_total
+    }
+  }
+`
+export const GET_TOTAL_CLAIMED_AND_UNCLAIMED_BY_ELECTION = gql`
```
Comment (leisterfrancisco) [inline webapp/src/gql/income.gql.js:118]: blank line

File: webapp/src/routes/IncomeReport/IncomePieChart.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+      </text>
+    </g>
+  )
+}
+
+const IncomePieChart = ({ data, coinType }) => {
+  const classes = useStyles()
+  const [activeIndex, setActiveIndex] = useState(0)
+
+  GLOBAL_COIN = coinType
```
Comment (leisterfrancisco) [inline webapp/src/routes/IncomeReport/IncomePieChart.js:93]: Let's try to avoid global variables

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

**Review by tetogomez — APPROVED** [review]
LGTM

File: .vscode/settings.json (hunk 1 above)
Comment (JAMares) [reply .vscode/settings.json:11]: This specifies how a directory works, on [eslint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) explain better

**Review by AngeloCG97 — APPROVED** [review]
LGTM

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (JAMares) [reply .vscode/settings.json:11]: Because an error appear whitout this

**Comment by JAMares** [comment]
Requested Changes fixed

**Review by tetogomez — APPROVED** [review]
LGTM

**Review by leisterfrancisco — APPROVED** [review]

**Pull request merged**